src/hotspot/share/asm/codeBuffer.hpp

```diff
@@ -546,6 +546,8 @@ class CodeBuffer: public StackObj DEBUG_ONLY(COMMA private Scrubber) {
     DEBUG_ONLY(: Scrubber(this, sizeof(*this)))
   {
     initialize_misc(name);
+    _total_start = 0;
+    _total_size  = 0;
```

Comment:
May be we should move this initialization from `initialize()` to `initialize_misc()` so you don't need to do this here.

Reply:
Otherwise following constructor also doesn't set them. `initialize(csize_t code_size, csize_t locs_size)` does not.

Reply:
Ok, moved.